R/fb_filter_species_traits.R

```diff
@@ -0,0 +1,258 @@
+#' Compute species coverage for each trait
+#' 
+#' @description
+#' For each trait computes the percentage of species without `NA` (missing 
+#' trait values).
+#' 
+#' @inheritParams fb_get_coverage
+#'
+#' @return A three-column `data.frame` with:
+#' - `trait`: the name of the trait;
+#' - `n_species`: the number of species with non-missing value for the trait;
+#' - `coverage`: the percentage of species with non-missing value for the trait.
+#' 
+#' @export
+#'
+#' @examples
+#' library("funbiogeo")
+#' 
+#' data("species_traits")
+#' 
+#' species_coverage_by_trait <- fb_count_species_by_traits(species_traits)
+
+fb_count_species_by_traits <- function(species_traits) {
```

Comment:
Maybe split the different functions in different files? It would increase the number of files but is easier to navigate.

Reply:
Done in #5aa6093d6b1ca7b259c30f2d5345db40ab69880f